Orange/data/table.py

```diff
@@ -11,7 +11,6 @@
 from tempfile import NamedTemporaryFile
 from urllib.parse import urlparse, unquote as urlunquote
 from urllib.request import urlopen
-from urllib.error import URLError
```

Comment:
> But it's unused

It just might have been meant for that. Indeed, doesn't really hurt if we just let urllib complain more verbosely than we would probably know to.

> it's a mess here.

![](http://vignette3.wikia.nocookie.net/uncyclopedia/images/4/4a/Orly_owl.jpg)